c2rust-analyze/src/borrowck/mod.rs

```diff
@@ -383,7 +384,7 @@ fn run_polonius<'tcx>(
 fn try_load_cached_output(facts_hash: &str) -> Option<Output> {
     let path = format!("polonius_cache/{}.output", facts_hash);
 
-    let f = File::open(&path).ok()?;
+    let f = BufReader::new(File::open(&path).ok()?);
```

Comment:
Is it faster to just read it all at once?  Sometimes I've found that to be faster.